packages/garbo-lib/src/wanderer/gingerbreadcity.ts

```diff
@@ -0,0 +1,31 @@
+import { availableAmount, haveOutfit, Location } from "kolmafia";
+import { DraggableFight, WandererFactoryOptions, WandererTarget } from "./lib";
+import { $item, $location, GingerBread } from "libram";
+
+export function gingerbreadFactory(
+  type: DraggableFight,
+  locationSkiplist: Location[],
+  options: WandererFactoryOptions,
+): WandererTarget[] {
+  if (
+    ["freefight", "freerun"].includes(type) &&
+    !locationSkiplist.some((loc) => GingerBread.LOCATIONS.includes(loc)) &&
```

Comment:
Is there a reason why we're going with this general a check instead of checking for the specific zone we're looking for?